pkg/tsdb/cloudwatch/log_query.go

```diff
@@ -95,24 +95,45 @@ func logsResultsToDataframes(response *cloudwatchlogs.GetQueryResultsOutput) (*d
 	}
 
 	frame := data.NewFrame("CloudWatchLogsResponse", newFields...)
+
+	bytesScanned := -1.0
+	recordsScanned := -1.0
+	recordsMatched := -1.0
```

Comment:
Why -1.0 as the default, as opposed to 0? I don't know the answer, asking as I'd like to find out :)

Reply:
I suppose my thinking was that a negative number makes it clear it's not a "valid" value to a user, but if this breaks some established convention I can change it to 0.

Reply:
If the statistic is not present, it should be omitted according to https://github.com/grafana/grafana-plugin-sdk-go/pull/191#discussion_r450278962

Or we could making a breaking SDK change to the QueryStat struct and make it a pointer (Which I am not against). The behavior as viewed by the frontend could remain the same, because this is in json so we could omitempty (which would omit when nil, but allow 0).